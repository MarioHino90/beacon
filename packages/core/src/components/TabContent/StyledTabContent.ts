/**
 * @fileoverview the styled tab content container
 */
import classNames from "classnames";
import styled from "styled-components";
import { getSpacing } from "@braze/beacon-styling";

import { StyledTabContentProps } from "./types";

export const StyledTabContent = styled.div.attrs(({ className }) => ({
  className: classNames(className, "bcl-tab-content"),
}))<StyledTabContentProps>`
  margin: ${getSpacing("xl")} 0;
`;
