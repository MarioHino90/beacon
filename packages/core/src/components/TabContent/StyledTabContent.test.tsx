import React from "react";
import { shallow } from "enzyme";
import theme from "@braze/beacon-theme";

import { StyledTabContent } from "./StyledTabContent";

describe("StyledTabContent", () => {
  it("should render correctly", () => {
    const wrapper = shallow(
      <StyledTabContent className="extraClass" theme={theme} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
