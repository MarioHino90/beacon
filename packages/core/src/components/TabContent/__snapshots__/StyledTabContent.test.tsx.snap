// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledTabContent should render correctly 1`] = `
.c0 {
  margin: 1.5rem 0;
}

<div
  className="c0 extraClass extraClass bcl-tab-content"
/>
`;
